pkgs/applications/graphics/megapixels/default.nix

```
@@ -0,0 +1,45 @@
+{ stdenv
+, lib
```

```suggestion
{ stdenv
```

Please only use one.

---

Why, is that a new packaging role? There's plenty of existing packages in Nixpkgs that use both for convenience.

```
↪ grep -rin "{ stdenv, lib," pkgs/ | wc -l
631
```

---

Gonna revert that change, see #108938, #108978